src/uopool/services/uopool.rs

```diff
@@ -160,9 +160,45 @@ where
 
     async fn remove(
         &self,
-        _request: tonic::Request<RemoveRequest>,
+        request: tonic::Request<RemoveRequest>,
     ) -> Result<Response<RemoveResponse>, tonic::Status> {
-        Err(tonic::Status::unimplemented("todo"))
+        let req = request.into_inner();
+
+        if let RemoveRequest {
+            hashes,
+            ep: Some(entry_point),
+        } = req
+        {
+            let entry_point = entry_point
+                .try_into()
+                .map_err(|_| tonic::Status::invalid_argument("invalid entry point"))?;
+            let mempool_id = mempool_id(&entry_point, &self.chain_id);
```

Comment:
I just want to say that these mempool id routing codes are duplicates in every method.  That's what I mean we need route service in https://github.com/Vid201/aa-bundler/issues/78#issuecomment-1483975167 so that we could remove dup codes here.

Reply:
We could do that later. Just want to bring up here.

Reply:
Yeah, I agree. The more features we add, the more I see the problem.

We have to definitely make that change later.